libs/dh/shared/ui-paginator/src/lib/dh-shared-ui-paginator.component.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, OnDestroy, inject, ViewChild, Input } from '@angular/core';
+import {
+  MatPaginator,
+  MatPaginatorIntl,
+  MatPaginatorModule,
+} from '@angular/material/paginator';
+import { TranslocoService } from '@ngneat/transloco';
+import { Subject, takeUntil } from 'rxjs';
+
+@Component({
+  standalone: true,
+  selector: 'dh-shared-ui-paginator',
+  imports: [MatPaginatorModule],
+  template: `
+    <mat-paginator
+      [length]="length"
+      [pageSize]="pageSize"
+      [pageSizeOptions]="pageSizeOptions"
+      [showFirstLastButtons]="true"
+      [attr.aria-label]="ariaLabel"
+    >
+    </mat-paginator>
+  `,
+})
+export class DhSharedUiPaginatorComponent implements OnDestroy {
+  @ViewChild(MatPaginator, { static: true }) private paginator!: MatPaginator;
+
+  @Input() length?: number;
+  @Input() pageSizeOptions: number[] = [50, 100, 150, 200, 250];
+  @Input() pageSize?: number;
```

Comment:
Question: Should this have a default value as well?

Reply:
Angular materials sets the `pagesize` to have a default value to 50, I will add it to make it explicit :) 
![image](https://user-images.githubusercontent.com/86852536/196868759-5386bc6a-9bf7-41c2-b80b-a777059b6676.png)